cli/cluster.go

```diff
@@ -100,11 +100,19 @@ func runClusterRemove(args *docopt.Args) error {
 	name := args.String["<cluster-name>"]
 
 	if config.Remove(name) {
+		msg := fmt.Sprintf("Cluster %q removed.", name)
+
+		// Select next available cluster as default
+		if len(config.Clusters) > 0 {
+			config.SetDefault(config.Clusters[0].Name)
```

Comment:
This logic should only be triggered if the cluster being removed is the default.